src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Validation/Result/HealthCertificateTechnicalValidationFailedViewModel.swift

```diff
@@ -9,61 +9,85 @@ struct HealthCertificateTechnicalValidationFailedViewModel: HealthCertificateVal
 		self.arrivalCountry = arrivalCountry
 		self.arrivalDate = arrivalDate
+		self.expirationDate = expirationDate
+		self.signatureInvalid = signatureInvalid
 	}
 
 	// MARK: - Internal
 
 	var dynamicTableViewModel: DynamicTableViewModel {
-		DynamicTableViewModel([
-			.section(
-				cells: [
-					.headlineWithImage(
-						headerText: AppStrings.HealthCertificate.Validation.Result.Failed.title,
-						image: UIImage(imageLiteralResourceName: "Illu_Validation_Invalid")
-					),
-					.footnote(
-						text: String(
-							format: AppStrings.HealthCertificate.Validation.Result.validationParameters,
-							arrivalCountry.localizedName,
-							DateFormatter.localizedString(from: arrivalDate, dateStyle: .short, timeStyle: .short),
-							DateFormatter.localizedString(from: Date(), dateStyle: .short, timeStyle: .short)
-						),
-						color: .enaColor(for: .textPrimary2)
-					),
-					.title2(text: AppStrings.HealthCertificate.Validation.Result.TechnicalFailed.subtitle),
-					.space(height: 10),
-					.headline(text: AppStrings.HealthCertificate.Validation.Result.TechnicalFailed.failedSectionTitle),
-					.body(text: AppStrings.HealthCertificate.Validation.Result.TechnicalFailed.failedSectionDescription),
-					.technicalFailedRulesCell(),
-					.textWithLinks(
-						text: String(
-							format: AppStrings.HealthCertificate.Validation.Result.moreInformation,
-							AppStrings.HealthCertificate.Validation.Result.moreInformationPlaceholderFAQ, AppStrings.Links.healthCertificateValidationEU),
-						links: [
-							AppStrings.HealthCertificate.Validation.Result.moreInformationPlaceholderFAQ: AppStrings.Links.healthCertificateValidationFAQ,
-							AppStrings.Links.healthCertificateValidationEU: AppStrings.Links.healthCertificateValidationEU
-						],
-						linksColor: .enaColor(for: .textTint)
-					)
-				]
+		 
+		var cells = [
+			DynamicCell.headlineWithImage(
+				headerText: AppStrings.HealthCertificate.Validation.Result.TechnicalFailed.title,
+				image: UIImage(imageLiteralResourceName: "Illu_Validation_Invalid")
+			),
+			.footnote(
+				text: String(
+					format: AppStrings.HealthCertificate.Validation.Result.validationParameters,
+					arrivalCountry.localizedName,
+					DateFormatter.localizedString(from: arrivalDate, dateStyle: .short, timeStyle: .short),
+					DateFormatter.localizedString(from: Date(), dateStyle: .short, timeStyle: .short)
+				),
+				color: .enaColor(for: .textPrimary2)
+			),
+			.title2(text: AppStrings.HealthCertificate.Validation.Result.TechnicalFailed.subtitle),
+			.space(height: 10),
+			.headline(text: AppStrings.HealthCertificate.Validation.Result.TechnicalFailed.failedSectionTitle),
+			.body(text: AppStrings.HealthCertificate.Validation.Result.TechnicalFailed.failedSectionDescription),
+			.technicalFailedRulesCell(failureText: AppStrings.HealthCertificate.Validation.Result.TechnicalFailed.notValidDateFormat, expirationDate: nil)
```

Comment:
```suggestion
			.body(text: AppStrings.HealthCertificate.Validation.Result.TechnicalFailed.failedSectionDescription)
```

We don't do the Schema check as the schema is always correct if the user was able to import their certificate.

Reply:
It's in Figma. Should I tell them they need to delete that section?

Reply:
Figma assumes that we check the schema and only show this message if the check fails. We check the schema when importing the certificate, therefore we don't have to check the schema again as it would always succeed. The means we never show the error that the schema check failed. Theoretically it could be removed from Figma as we will never show it.

Reply:
Done